File: src/main/java/org/jcodec/containers/mp4/WaveExtBoxes.java
package org.jcodec.containers.mp4;

import org.jcodec.containers.mp4.boxes.EndianBox;
import org.jcodec.containers.mp4.boxes.FormatBox;

public class WaveExtBoxes extends Boxes {
    public WaveExtBoxes() {
        mappings.put(FormatBox.fourcc(), FormatBox.class);
        mappings.put(EndianBox.fourcc(), EndianBox.class);
        //            mappings.put(EsdsBox.fourcc(), EsdsBox.class);
    }
}

File: src/main/java/org/jcodec/containers/mp4/WaveExtBoxes.kt
package org.jcodec.containers.mp4

import org.jcodec.containers.mp4.boxes.EndianBox
import org.jcodec.containers.mp4.boxes.FormatBox

class WaveExtBoxes : Boxes() {
    init {
        mappings[FormatBox.fourcc()] = FormatBox::class.java
        mappings[EndianBox.fourcc()] = EndianBox::class.java
        //            mappings.put(EsdsBox.fourcc(), EsdsBox.class);
    }
}